### DISCUSSION TITLE
Display LCD I2C - text switch off at DisplayText Hello Tasmota

### DISCUSSION THREAD

**Discussion by rapi3** [arendst/Tasmota#11321] [Q&A] [answered]
Hello,
I have a display LCD with I2C driver - 2 rows / 16 colons connected to wemos D1 mini with custom compiled Tasmota 9.3.1.1 ( latest development and also tested with official Tasmota-display.bin )
#define USE_I2C
#define USE_DISPLAY
#define USE_DISPLAY_LCD
#define SHOW_SPLASH
{"Display":{"Model":1,"Width":16,"Height":2,"Mode":0,"Dimmer":1,"Size":1,"Font":1,"Rotate":0,"Refresh":2,"Cols":[16,8],"Rows":2}}

I use it to display date & time on first line and temp from 3 sensors on second line.
My problem is that If I write anything for example DisplayText Hello or at any text change display will switch off - text will disappear light stay on and I have to write DisplayText [O] or Power1 1 or DisplayDimmer 100 so the text will appear on LCD.

Is this the normal driver behavior when writing to this LCD display or something I do it is wrong because this introduce flicker after any writing and I don't think it is normal.

p.s.
after DisplayDimmer 100 the result it is 15 ??
22:35:04.260 CMD: displaydimmer 100
22:35:04.269 MQT: Wemos/stat/W12/RESULT = {"DisplayDimmer":15}

thank you

**Comment by ascillato2** [comment, marked as answer]
Please, address this to the [Tasmota Support Chat](https://discord.gg/Ks2Kzd4). The chat is a better and more dynamic channel for helping you.Thanks.

---------------------------

### Support Information 
See [Docs](https://tasmota.github.io/docs/) for more information.
See [Chat](https://discord.gg/Ks2Kzd4) for more user experience.
See [Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
